src/components/locale-selector/LocaleSelector.test.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from "react";
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import LocaleSelector from "./LocaleSelector";
+import { ISettings } from "@/AppSettings";
+
+test("should render button to change locale", () => {
+  render(<LocaleSelector />);
+  expect(screen.getByText(/change language/i)).toBeInTheDocument();
```

Comment:
Firstly, I don't think the name of the test and what this is testing line up very will.

Secondly, I'd be tempted to move this assertion into the other test straight after the render, then test it changes when the language changes (assuming it is meant to change).

Reply:
Sound good. I've updated test and did assertion in this way only. Thanks